addons/l10n_be_edi/models/ir_actions_report.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+
+
+class IrActionsReport(models.Model):
+    _inherit = 'ir.actions.report'
+
+    @api.model
+    def postprocess_pdf_report(self, res_id, pdfreport_path, attachment_name):
+        attachment_id = super(IrActionsReport, self).postprocess_pdf_report(res_id, pdfreport_path, attachment_name)
+        if attachment_id and self.model == 'account.invoice':
+            invoice_id = self.env[self.model].browse(res_id)
```

Comment:
Because `model` is a field on the object, it's not the current model (which would be `_name`).